pydocx/export/html.py

```diff
@@ -361,6 +361,114 @@ def export_paragraph_property_indentation(self, paragraph, results):
 
         return results
 
+    def get_previous_level_paragraph(self, num_id, level_id):
+        level_id = int(level_id)
+
+        while True:
+            if level_id == 0:
+                prev_level_id = level_id
+            else:
+                prev_level_id = level_id - 1
+
+            prev_level_paragraphs = self.numbering_level_listing_track[num_id][prev_level_id]
+            if prev_level_paragraphs:
+                return prev_level_paragraphs[-1]
+
+            if prev_level_id == 0 and not prev_level_paragraphs:
+                # this is an edge case with older version of word when it may contain a sublist
+                # into a separate num_id.
+                break
+
+            level_id -= 1
+
+        return None
+
+    def export_listing_paragraph_property_indentation(self, paragraph, level_properties,
```

Comment:
Sorry that I wasn't clear. Like this:

```python
def export_listing_paragraph_property_indentation(
    self, 
    paragraph, 
    level_properties, 
    include_text_indent=False,
):
```

Actually, don't worry about these minor style issues, including my notes on comment capitalization/punctuation. I can handle all of that trivial stuff after we get this merged in. We can't expect you to read our mind on those things, since they aren't documented in our [coding standards](https://pydocx.readthedocs.io/en/latest/development.html#coding-standards).